src/Employer/Employer.Web/Views/Shared/_Layout.cshtml

```diff
@@ -197,6 +197,7 @@ public bool IsErrorPage()
     <script src="/lib/esfa-govuk/assets/javascripts/govuk-template.js"></script>
     <script src="/lib/esfa-govuk/assets/javascripts/govuk/show-hide-content.js"></script>
     <script src="/lib/esfa-stacker/stacker.js"></script>
+    <script src="/lib/tinymce-4.9.2/tinymce.min.js"></script>
```

Comment:
Does this need to be on every page? Is it the dependency that application.js has on it which requires it here?

Reply:
👍 I'll only reference it in the required pages. application.js is happy enough as long as we don't call initializeEditor without the script being referenced.